raster/r.kappa/prt_kappa.c

```diff
@@ -0,0 +1,57 @@
+#include <stdlib.h>
+#include <grass/gis.h>
+#include <grass/glocale.h>
+#include "kappa.h"
+#include "local_proto.h"
+
+
+void prt_kappa(void)
+{
+    int i;
+    FILE *fd;
+
+    if (output == NULL)
+        fd = stdout;
+    else if ((fd = fopen(output, "a")) == NULL) {
+        G_fatal_error(_("Cannot open file <%s> to write kappa and relevant parameters"),
+                      output);
+        return;
+    }
+
+    /* print out the comission and omission accuracy, and conditional kappa */
+    fprintf(fd, "\nCats\t%% Comission\t%% Omission\tEstimated Kappa\n");
+    for (i = 0; i < ncat; i++) {
+        fprintf(fd, "%ld\t", rlst[i]);
+        if (metrics->user_acc[i] == -999)
```

Comment:
Perhaps make a constant for `-999` in `kappa.h`?

Reply:
I was thinking something like `static const double na_value = -999.0;` not a `#define`.

Reply:
Good idea.